client/src/components/ApplicantsForm.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from "react";
```

Comment:
This form is not what the client has requested, create the  required form and we can work on the fetch request together when that is complete.